core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -132,7 +134,8 @@ object TestUtils extends Logging {
   /**
    * Create a kafka server instance with appropriate test settings
    * USING THIS IS A SIGN YOU ARE NOT WRITING A REAL UNIT TEST
-   * @param config The configuration of the server
+    *
+    * @param config The configuration of the server
```

Comment:
Looks like IntelliJ reformatted the code here (and a few other places in this file). It's quite annoying when it does that.